Workflows/Extensions/ToUnityITI.cs

```diff
@@ -0,0 +1,29 @@
+using Bonsai;
+using System;
+using System.ComponentModel;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reactive.Linq;
+using NetMQ;
+using OpenCV.Net;
+
+[Combinator]
+[Description("Get SecondsInterTrialInterval. From ZeroMQ Unity ITI Message")]
+[WorkflowElementCategory(ElementCategory.Transform)]
+public class ToUnityItI
```

Comment:
```suggestion
public class ToUnityIti
```

The class name is not consistent with the file name, and also not consistency with acronym naming conventions.

Reply:
Fixed in d47ab3ace08e6d5dc506cb66badafc8fb75176d6